project/Release.scala

```diff
@@ -59,6 +59,6 @@ object Release extends Build {
     }
 
   def javaVersionCheckSettings = Seq(
-    javaVersionPrefix in javaVersionCheck := Some("1.6")
+    javaVersionPrefix in javaVersionCheck := Some("1.7")
```

Comment:
I know, sorry, that change is in a temporary commit to help testing (don't have jdk 6 on my laptop anymore)